builder/amazon/ebssurrogate/step_register_ami.go

```diff
@@ -25,6 +25,8 @@ func (s *StepRegisterAMI) Run(state multistep.StateBag) multistep.StepAction {
 
 	ui.Say("Registering the AMI...")
 
+	s.BlockDevices = config.BlockDevices.BuildAMIDevices()
```

Comment:
this is already set here https://github.com/hashicorp/packer/blob/7ff73a9d47d67344541969ef24aa30c7162ef2ee/builder/amazon/ebssurrogate/builder.go#L212